spec/constants_for_spec.rb

```diff
@@ -0,0 +1,87 @@
+COMMON_PHRASES = {
```

Comment:
All these constants you have in I18n.t
Why you don't reuse I18n.t?

Reply:
Replaced with I18n